config/system.php

```diff
@@ -407,6 +407,14 @@
 'default_value'=>'y'
 );
 
+$system__options_general[]=array(
+'option_name'=>'include_weekday_in_session_name',
```

Comment:
Actually, would rather keep it that way - for the moment. All system options in that file are currently formatted this way. If we were to change this formatting, then I would do it in a housekeeping branch all at once.